lib/rmt/cli/clean.rb

```diff
@@ -121,18 +121,15 @@ def dangling_packages_list(repo_base_dir, repomd_file)
     packages = (actual_packages - expected_packages).sort
 
     packages.map do |file|
-      next nil unless File.exist?(file)
-
-      # Only remove files if they were not recently created
-      file_stat = File.stat(file)
-      next nil if (Time.current - file_stat.mtime) < DANGLING_FILE_MINIMUM_AGE
+      file_status = File.stat(file)
```

Comment:
Do you need to rename this variable? `file_stat` makes it clear that it's a Stat object whereas `file_status` is far less definite.

Reply:
Reverted it to the original naming on 020f16f.